ngi_visualizations/fpkm_scatter/fpkm_scatter.py

```diff
@@ -0,0 +1,253 @@
+#!/usr/bin/python
+"""
+fpkm_scatter.py
+
+Takes FPKM counts from two conditions and makes a scatter plot.
+Also calculates a r-squared correlation score.
+
+Either takes two Cufflinks FPKM summary files or a summary file
+with multiple samples.
+"""
+
+from __future__ import print_function
+
+import argparse
```

Comment:
Every other script in the package also uses argparse, so I think I'll leave it either way..